pkgs/development/python-modules/confuse/default.nix

```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  isPy3k,
+  pyyaml
+}:
```

Comment:
although kind of weird, generally the style is like:
```
{ lib
, buildPythonPackage
, fetchPypi
....
```
helps to save on diff noise, as it's now 1 line = 1 change